Cateye style for Gamepad

skybaks opened the pull request:
![image](https://user-images.githubusercontent.com/52106022/135769355-9fde1cc4-fe9f-48ef-be31-8f5d3a2bb160.png)

This adds a new style of gamepad visualization per my interpretation of #33 .

During implementation I discovered that nanoVG does not want to fill a triangle with the one inflected edge, so I had to split each into two triangles.

For settings, I used the settings from Uniform with the exception of border color and size. I initially didnt want to duplicate a bunch of code to restroke the outlines for a border but if you think it's useful I can work on moving the path code to a function and stroke a border.

codecat commented:
This looks awesome! I'll be able to properly review this tomorrow.

At first sight, the steering arrow fills aren't the same as they are on that PadViz version, so perhaps that should be changed? Not sure what people would prefer though.

skybaks commented:
Oooh, does the steer on padvis stretch the shape from the inflected edge? Hm, I could work on switching to that.

codecat commented:
Yeah, it does. I've never liked that style, but it's what that specific padviz is doing. Maybe there can be an option to toggle between those 2 modes.

skybaks commented:
Can I do that with the linear gradient fill method, or do you think I would need to fill another path?

codecat commented:
You can do it without any gradient, you just need to draw 2 triangles where its most right/left point is controlled by the value of steering.

skybaks commented:
![image](https://user-images.githubusercontent.com/52106022/135940809-036dd051-d1a3-4cfc-b369-da28e525413d.png)

Updated to use your suggestion for steering visual. I also moved all the path code to a reusable function: `FillInflectedTriangle()` and then pulled in the settings for border and stroked that from `StrokeInflectedTriangle()`.

Btw, "Inflected Triangle" is just some name I came up with on my own. I don't know what it should really be called but I am more than happy to change those if you would like a better name.

skybaks commented:
Per your earlier suggestion, I added a boolean setting to toggle between the "stretchy" and simple steering visualization.
What are your thoughts on the setting's wording and default setting?

codecat commented:
The defaults are okay! Thanks for the PR! :D
